man/systemd-activate.xml

```diff
@@ -104,6 +105,13 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>-d</option></term>
+        <term><option>--datagram</option></term>
+
+        <listitem><para>Listen to datagram sockets.</para></listitem>
```

Comment:
In this new version, I think there's need for some more text here or in the hunk above, explaining that by default, the socket will be opened in stream mode, but can be turned into datagram mode with this flag.